### PR TITLE
ECL-359: Update for returns submitted page

### PR DESCRIPTION
**Pull request by jamesvarghese701**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/pages/returns/ReturnsPage.scala [hunk 1]
```diff
@@ -171,4 +171,21 @@ object ReturnsPage extends BasePage {
...
+    val actual = getText(
+      By.xpath(s"//p[contains(text(),'ECL return number: ')]")
+    )
+    assert(actual.contains(value))
+    this
+  }
+
+  def assertAmountNeedToPay(value: String): this.type = {
+    val actual = getText(
+      By.xpath(s"//p[contains(text(),'Amount you need to pay: £10,000')]")
```
Comment (TonyCOgilvie) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/returns/ReturnsPage.scala:186]: Why is the amount fixed at £10,000? Is this the only liability scenario tested? Would it not be more flexible to simply match on the text excluding the amount (like is done for the ECL return number in the above check)?

Comment (jamesvarghese701) [reply src/test/scala/uk/gov/hmrc/test/ui/pages/returns/ReturnsPage.scala]: done

File: src/test/resources/features/Returns.feature [hunk 2]
```diff
@@ -121,47 +122,47 @@ Feature: Submit ECL Return
...
     Then I should be on the page that says <Expected content>
     Examples:
-      | UK Revenue | Expected content                                                        |
-      | 10200000   | The amount of levy you need to pay for this financial year is £10,000.  |
-      | 35999999   | The amount of levy you need to pay for this financial year is £10,000.  |
-      | 36000000   | The amount of levy you need to pay for this financial year is £36,000.  |
-      | 999999999  | The amount of levy you need to pay for this financial year is £36,000.  |
-      | 1000000000 | The amount of levy you need to pay for this financial year is £250,000. |
+      | UK Revenue | Expected content                                           |
+      | 10200000   | You need to pay £10,000 for the 2022-2023 financial year.  |
```
Comment (TonyCOgilvie) [inline src/test/resources/features/Returns.feature:126]: 2022-2023 is dynamic based on the financial year of the obligation that the return is for, so does it make sense to include it like this (I don't know if you can make it dynamic)?

**Review by TonyCOgilvie — APPROVED** [review]

File: src/test/resources/features/Returns.feature (hunk 2 above)
Comment (TonyCOgilvie) [reply src/test/resources/features/Returns.feature:126]: Discussed via Slack

**Pull request merged**